src/fragments/lib/datastore/js/getting-started/30_platformIntegration.mdx

```diff
@@ -14,4 +14,72 @@ Start with [Create React app](https://create-react-app.dev):
 npx create-react-app amplify-datastore --use-npm
 cd amplify-datastore
 npx amplify-app@latest
+```  
+
+</Block>
```

Comment:
Is this a typo? There isn't an opening block statement for it, or maybe it just needs to be deleted?

Reply:
@timngyn - please do! I think it is a typo for sure

Reply:
I've had a look, and it seems like there's an open tag without a closing tag as well. @abdallahshaban557, mind having a look? I want to make sure we aren't scrobbling this up

Reply:
That one can also be removed - I'll submit an update shortly

Reply:
Updated now